custom-service-adapter/README.md

```diff
@@ -0,0 +1,19 @@
+# Custom Service Adatper with Knot.x
+
+In this tutorial we create a page: [http://localhost:8092/html/books.html](http://localhost:8092/html/books.html) books from database.
+Database access is not available via WebAPI thus we have 2 options:
+
+1. integrate with data source implementing WebAPI layer
+2. write simple [Knot.x Service Adapter](https://github.com/Cognifide/knotx/wiki/ServiceAdapter)
+
+Before running this example you will need to setup HSQL database.
```

Comment:
For the demonstration purposes we're going to use HSQL database in this example.